### ISSUE TITLE
Exponential numbers like 1e-06 produce "Dimension too large"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halamalala/inkscape2tikz#34]
```
Very small numbers are writen in exponential number fromat like 1e-06. For 
example in

\path[shift={(1e-06,0.03057816)},fill=black] ... ;

or

\begin{scope}[cm={{1.0,-0.43924987,0.0,1.0,(-2.3578e-06,37.193992)}}] ... 

this format can't interpreted and will produce a "Dimension too large." and a 
Fatal error during the compiling. When replacing 1e-06 to 0.000001 it works.

TeX-version:
pdfTeX, Version 3.1415926-1.40.10 (TeX Live 2009/Debian)

Please provide any additional information below.


```

Original issue reported on code.google.com by `beppoAnm...@googlemail.com` on 9 Sep 2011 at 1:28

Attachments:
- [Test.svg](https://storage.googleapis.com/google-code-attachments/inkscape2tikz/issue-34/comment-0/Test.svg)
- [Test.tex](https://storage.googleapis.com/google-code-attachments/inkscape2tikz/issue-34/comment-0/Test.tex)

**Comment by GoogleCodeExporter** [comment]
Original comment by `kjel...@gmail.com` on 12 Jul 2012 at 8:49
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision ff35c7940518.
```

Original comment by `kjel...@gmail.com` on 14 Jul 2012 at 5:19
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I have fixed the bug, but the figure is still not rendered properly. The reason 
is that  PGF does not scale line widths by default. Not sure how to handle this.
```

Original comment by `kjel...@gmail.com` on 14 Jul 2012 at 5:21
